SPECS/patterns-ceph-containers/patterns-ceph-containers.spec

```diff
@@ -0,0 +1,102 @@
+Summary:        Patterns for the Ceph containers
+Name:           patterns-ceph-containers
+Version:        1.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Metapackages
+URL:            http://en.opensuse.org/Patterns
+Source0:        %{name}-rpmlintrc
+ExclusiveArch:  x86_64 aarch64 ppc64le s390x
+
+%description
+This is an internal package that is used to create the patterns as part
+of the installation source setup. Installation of this package does
+not make sense.
+
+%package ceph_base
+Summary:        Ceph base
+Group:          Metapackages
+Provides:       pattern() = ceph_base
+#Provides:      pattern-icon() = yast-iscsi-client
+Provides:       pattern-category() = Containers
+Provides:       pattern-order() = 3000
+Provides:       pattern-visible()
+Requires:       ca-certificates
+Requires:       ceph
+Requires:       ceph-base
+Requires:       ceph-common
+Requires:       ceph-fuse
+Requires:       ceph-grafana-dashboards
+Requires:       ceph-mds
+Requires:       ceph-mgr
+Requires:       ceph-mgr-cephadm
+Requires:       ceph-mgr-dashboard
+Requires:       ceph-mgr-rook
+#Temporarily commenting out dependency to allow package to build
+#Requires:      ceph-mgr-diskprediction-local
+Requires:       ceph-mon
+Requires:       ceph-osd
+Requires:       ceph-prometheus-alerts
+Requires:       ceph-radosgw
+Requires:       cephadm
+Requires:       e2fsprogs
+Requires:       gptfdisk
+Requires:       kmod
+Requires:       lvm2
+#Temporarily commenting out dependency to allow package to build
+#Requires: ceph-iscsi
+Requires:       rbd-mirror
+Requires:       rbd-nbd
+
+%description ceph_base
+This provides the base for the Ceph, Rook, Ceph CSI driver packages and containers.
+
+%prep
+# empty on purpose
+
+%build
+# empty on purpose
+
+%install
+mkdir -p %{buildroot}%{_docdir}/packages/patterns-ceph-containers/
+echo 'This file marks the pattern ceph-base to be installed.' >%{buildroot}%{_docdir}/packages/patterns-ceph-containers/ceph_base.txt
+
+%files ceph_base
```

Comment:
I think it's fine to keep this package, but I'd remove the files completely, since they seem to be useless. An empty `%files ceph_base` section is completely acceptable. See `core-packages.spec` for a similar example of.